bundle.js

```diff
@@ -0,0 +1,501 @@
+	chance17.action = function(player) {
+		player.money += 100;
+	};
+
+	var chanceDeck = [chance1, chance2, chance3, chance4, chance5, chance6, chance7, chance8, chance9, chance10, chance11, chance12, chance13, chance14, chance15, chance16, chance17];
+
+
+
+
+	// http://stackoverflow.com/questions/6274339/how-can-i-shuffle-an-array-in-javascript
+	function shuffle(o){
+	    for(var j, x, i = o.length; i; j = Math.floor(Math.random() * i), x = o[--i], o[i] = o[j], o[j] = x);
+	    return o;
+	}
+
+	var monopoly = Object.create(Game);
+
+	monopoly.players = players;
+	monopoly.chanceCards = chanceDeck;
+	monopoly.communityChestCards = communityChestDeck;
+	shuffle(monopoly.chanceCards);
+	shuffle(monopoly.communityChestCards);
+	monopoly.randomizeCurrentPlayer();
+
+	n = 0;
+	while (n < 100) {
+		console.log(monopoly.players[monopoly.currentPlayer]);
+		monopoly.movePlayer();
+		console.log(monopoly.dice);
+		monopoly.nextPlayer();
+		n += 1;
+	}
+
+/***/ },
+/* 1 */
+/***/ function(module, exports) {
+
+	var Player = {
+	    name: "",
+	    money: 500,
+	    properties: [],
+	    position: 0,
+	    hasGetOutFreeCard: false,
+	    inJail: false,
+	    turnsInJail: 0,
+
+	    payBail: function() {
+	        this.releaseFromJail();
+	        this.money -= 50;
+	    },
+
+	    getOutOfJailFree: function() {
+	        this.releaseFromJail();
+	        this.hasGetOutFreeCard = false;
+	    },
+
+	    releaseFromJail: function() {
+	        this.inJail = false;
+	        this.turnsInJail = 0;
+	    },
+
+	};
+
+	module.exports = Player;
+
+/***/ },
+/* 2 */
+/***/ function(module, exports) {
+
+	var Card = {
+	    text: "",
+	    action: null
+	};
+
+	module.exports = Card;
+
+/***/ },
+/* 3 */
+/***/ function(module, exports) {
+
+	var Game = {
+	    players: [],
+	    currentPlayer: 0,
+	    board: {},
+	    dice: [],
+	    chanceCards: [],
+	    communityChestCards: [],
+	    doublesCount: 0,
+
+	    randomizeCurrentPlayer: function() {
+	        this.currentPlayer = Math.floor(Math.random() * this.players.length);
+	    },
+
+	    rollDice: function(numberOfDice, sides) {
+	        if (!sides) {
+	            sides = 6;
+	        }
+	        this.dice = [];
+	        for (i = 0; i < numberOfDice; i++) {
+	            roll = Math.floor(Math.random() * sides) + 1;
+	            this.dice.push(roll);
+	        }
+	    },
+
+	    movePlayer: function() {
+	        this.rollDice(2);
+	        if (this.players[this.currentPlayer].inJail) {
+	            this.players[this.currentPlayer].turnsInJail += 1;
+	            if (this.isDoubles(this.dice)) {
+	                this.players[this.currentPlayer].releaseFromJail();
+	                this.move();
+	            } else if (this.players[this.currentPlayer].turnsInJail === 3) {
+	                this.players[this.currentPlayer].payBail();
+	                this.move();
+	            }
+	        } else {
+	            this.move();
+	        }
+	    },
+
+	    move: function() {
```

Comment:
might be easier to abstract this to take a dice roll argument so we can call move(diceRoll) and not have to refer to this.dice